apis/k8ssandra/v1alpha1/cassandraconfig_types.go

```diff
@@ -200,7 +200,7 @@ type JvmOptions struct {
 	// Cass Config Builder: supported for Cassandra 4.0 in jvm11-server.options.
 	// Corresponds to: -Dio.netty.tryReflectionSetAccessible=true.
 	// +optional
-	NettyTryReflectionSetAccessible *bool `json:"netty_try_reflection_set_accessible,omitempty" cass-config:">=4.x:jvm11-server-options/io_netty_try_reflection_set_accessible;dse@>=6.8.x,hcd@>=1.x.x:jvm8-server-options/io_netty_try_reflection_set_accessible"`
```

Comment:
the heap size isn't affected by this since it ends up in jvm-server.options, and not a jdk specific file.

We won't e2e test all of the possible jvm options, that would be overkill, so I'd rather keep the check for the heap size specifically as it breaks things in a big way.
I reckon we may have bugs in how some of them are processed.